Source/WebKit/UIProcess/WebFramePolicyListenerProxy.h

```diff
@@ -45,6 +47,8 @@ enum class ShouldExpectAppBoundDomainResult : bool { No, Yes };
 enum class ShouldWaitForInitialLinkDecorationFilteringData : bool { No, Yes };
 enum class WasNavigationIntercepted : bool { No, Yes };
 
+static constexpr Seconds defaultSafeBrowsingTimeout { 2_s };
```

Comment:
I think this is not enough for many network configurations.  The time should be based on how much time the request roundtrip to the server took.